dbms/src/Common/FailPoint.cpp

```diff
@@ -98,13 +98,18 @@ std::unordered_map<String, std::shared_ptr<FailPointChannel>> FailPointHelper::f
     M(pause_when_altering_dt_store)          \
     M(pause_after_copr_streams_acquired)
 
+#define APPLY_FOR_RANDOM_FAILPOINTS_ENABLED_OUTSIDE(M) \
+    M(random_tunnel_failpoint)                         \
+    M(random_receiver_failpoint)
+
```

Comment:
Maybe you should declare these two failpoints in `APPLY_FOR_FAILPOINTS`, or you can not get them enabled by using `FailPointHelper::enableFailPoint`

https://github.com/pingcap/tiflash/blob/efd45579a2d8b5d82a4696ad2c58e81119deaa3a/dbms/src/Common/FailPoint.cpp#L158-L174